arlo-client/src/components/Audit/Setup/Review/index.tsx

```diff
@@ -108,12 +88,38 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
         return
       }
       refresh()
+      history.push(`/election/${electionId}/progress`)
     } catch (err) /* istanbul ignore next */ {
       // TEST TODO
       toast.error(err.message)
     }
   }
 
+  const [openConfirmDialog, confirmDialog] = useConfirmLaunch(submit)
```

Comment:
I don't think this needs to be a hook to be abstracted out. I think a more common pattern would be to have the state controlled in this component:

```ts
const [isConfirmDialogOpen, setConfirmDialogOpen] = useState<boolean>(false)
```
and then rework the `useConfirmLaunch` hook into a component `ConfirmLaunchDialog` that takes props `isOpen: bool`, `onClose: () => void`, and `onLaunch: () => void`.

So you would render:

```ts
<ConfirmLaunchDialog isOpen={isConfirmDialogOpen} onLaunch={submit} onClose={() => setIsConfirmDialogOpen(false)} />
```

Reply:
@jonahkagan True, that. I think I honestly accidentally made it into a hook, haha.